qmlbind/src/metaobject.cpp

```diff
@@ -6,6 +6,25 @@
 
 namespace QmlBind {
 
+// See qmetaobject_p.h for flag details
```

Comment:
Today it came to my mind that a hint where to look all this up would be nice as it's obviously not in the Qt documentation, but you already added that in the meantime, nice. :smile: